sdk/src/main/resources/reference.conf

```diff
@@ -38,7 +38,16 @@ akkaserverless {
         internal-dispatcher = akka.actor.default-dispatcher
       }
 
-      coordinated-shutdown.exit-jvm = on
+      coordinated-shutdown {
+        exit-jvm = on
+        exit-code = 0
+
+        reason-overrides {
+          "com.akkaserverless.javasdk.AkkaServerlessRunner$BindFailure$" {
+            exit-code = -1
```

Comment:
ideally we would swap this around, and fail unless everything explicitly succeeded successfully, but I wanted to take one step at a time